.circleci/test-in-cluster.sh

```diff
@@ -78,10 +78,12 @@ KIND_CONFIG="${CWD}/kind-config.yaml"
     kubectl cp sidecar:/tmp/hello.world /tmp/hello.world
     kubectl cp sidecar:/tmp/cm-kubelogo.png /tmp/cm-kubelogo.png
     kubectl cp sidecar:/tmp/secret-kubelogo.png /tmp/secret-kubelogo.png
+    kubectl cp sidecar:/tmp/script_result /tmp/script_result
 
     echo -n "Hello World!" | diff - /tmp/hello.world \
       && diff ${CWD}/kubelogo.png /tmp/cm-kubelogo.png \
-      && diff ${CWD}/kubelogo.png /tmp/secret-kubelogo.png
+      && diff ${CWD}/kubelogo.png /tmp/secret-kubelogo.png \
+      && diff - /tmp/script_result
```

Comment:
`-` is for stdin which is empty only by coincidence it seems we're expecting an empty file here so how about using the empty string explicitly?

Reply:
my mistake, will correct that to check only if the file exists